docs/releases/3.0.txt

```diff
@@ -223,7 +223,9 @@ Features deprecated in 3.0
 Miscellaneous
 -------------
 
-* ...
+* To limit creation of sessions and hence favor some caching strategies, Django
```

Comment:
I'm a little confused about this note. This reads like a backwards incompatible change but isn't the language still stored in the session for now? That said, I wonder if some of the changes like the removals in `auth.logout()` merit a mention under "backwards incompatible changes"?